app/controllers/Application.scala

```diff
@@ -45,8 +45,16 @@ class Application(
     Redirect(location + path, request.queryString)
   }
 
-  def bundleLanding(title: String, id: String, js: String, newDesigns: Boolean): Action[AnyContent] = CachedAction() { implicit request =>
-    if (newDesigns) {
+  def bundleLanding(title: String, id: String, js: String, newDesigns: String): Action[AnyContent] = CachedAction() { implicit request =>
+    if (newDesigns == "circles") {
+      Ok(views.html.bundleLanding(
+        title,
+        "support-landing-page-old",
+        "supportLandingPageOld.js",
+        contributionsPayPalEndpoint,
+        description = Some(stringsConfig.bundleLandingDescription)
+      ))
+    } else if (newDesigns == "circles-garnett") {
```

Comment:
The duplication in the 3 cases can be reduced. It looks like the only differences between them are the values of `id` and `js`.

Reply:
Yup, it's only temporary though. We'll be getting rid of this query param soon 🤞